multiregion/scripts/permanent-fallback.sh

```diff
@@ -0,0 +1,21 @@
+echo -e "\n==> Switching replica placement constraints for multi-region-default\n"
+
+docker-compose exec broker-east-3 kafka-configs \
+	--bootstrap-server broker-east-3:19093 \
+	--alter \
+	--topic multi-region-default \
+	--replica-placement /etc/kafka/demo/placement-multi-region-async-reverse.json
```

Comment:
There is a mismatch: the topic is `multi-region-default` but the placement file is for `multi-region-async`